src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -114,11 +104,7 @@ bool TryParseUri(string u, out Uri uri)
 
 		protected double ParseMetricsInterval(ConfigurationKeyValue kv)
 		{
-			string value;
-			if (kv == null || string.IsNullOrWhiteSpace(kv.Value))
-				value = ConfigConsts.DefaultValues.MetricsInterval;
-			else
-				value = kv.Value;
+			var value = kv == null || string.IsNullOrWhiteSpace(kv.Value) ? ConfigConsts.DefaultValues.MetricsInterval : kv.Value;
```

Comment:
Was this a suggestion based on `.editorconfig`? To me the original one was way more readable, but this is just a nit.

Reply:
It's a VS or ReSharper suggestion. It's better because you're only assigning `value` once in the code. It's more robust. The operands of the ternary operator could go on separate lines, but I don't see how this would make it more readable for such short code.